gamemodes/core/commands/cmds_bail.inc

```diff
@@ -0,0 +1,15 @@
+CMD:bail(playerid) {
+    if(PlayerJailed[playerid] == false) {
+        return SendErrorMsg(playerid, "You're not jailed!");
+    }
+    if(timeLeftCounter[playerid] != 0) {
+        return SendErrorMsg(playerid, "You've not served your sentence!");
+    }
+    if(GetPlayerMoney(playerid) < BailBalance[playerid]) {
+        return SendErrorMsg(playerid, "You don't have enough money to pay 5000");
```

Comment:
static value, format it instead to match the value of BailBalance.

Reply:
Done